scripts/generate-sdk/languages/go.sh

```diff
@@ -149,7 +154,7 @@ generate_go_sdk() {
             --git-user-id ${GIT_USER_ID} \
             --git-repo-id ${GIT_REPO_ID} \
             --global-property apis,models,modelTests=true,modelDocs=false,apiDocs=false,supportingFiles \
-            --additional-properties=isGoSubmodule=true,enumClassPrefix=true
+            --additional-properties=isGoSubmodule=true,enumClassPrefix=true,foo=bar,$regional_api
```

Comment:
I think you missed to remove the foo=bar value
```suggestion
            --additional-properties=isGoSubmodule=true,enumClassPrefix=true,$regional_api
```

Reply:
Hupps

Reply:
Removed